packages/shared/src/logger.ts

```diff
@@ -51,13 +51,14 @@ class Logger {
     this.sessionId = randomUUID();
   }
 
+  // This should be called with the name once at the entry point.
+  // For example, with the Playwright plugin, it is called in the Reporter interface constructor.
   initialize(app: string, version: string | undefined) {
     if (this.initialized) {
       console.warn(`Logger already initialized.`);
     }
 
     this.initialized = true;
-    this.localDebugger = dbg(app);
```

Comment:
Before landing this PR I just need to know what our DEBUG namespace should be called. Do we settle on `replayio` for everything? or `replay`?

Reply:
"replayio" is our CLI package name so IMO it would be a little confusing to use that for everything